deployment-examples/aws/k8s/README.md

```diff
@@ -34,19 +31,23 @@ Once all the prerequisites have been installed and created, update the repositor
 
 For any changes for cronjob, configure the properties in [values.yaml](./values.yaml).
 
-## Create release.
+## Creating AWS cluster (optional)
```

Comment:
So you don't need a cluster to deploy to AWS?

Reply:
https://github.com/commercetools/commercetools-subscriptions/pull/48/commits/3041202de8f13db3d80bb37b0ecca11c6888d5f4

It is needed only for the first time.
I corrected the sentence.